src/types/types.ts

```diff
@@ -454,7 +454,12 @@ export type EdgeObjectTemplate = Array<
 export interface EdgeCurrencyInfo {
   // Basic currency information:
   readonly pluginId: string
+
+  /** deprecated: refers to chain, use chainDisplayName instead */
   displayName: string
+
+  chainDisplayName?: string
+  assetDisplayName?: string
```

Comment:
Please make these two mandatory, not optional. That way the GUI can use them reliably. Yes, it's maybe a bit annoying for the plugins on the next core upgrade, but we consider this a non-breaking change according to our plugin compatibility policy, since things still work at run-time.